src/components/home/Header/Header.tsx

```diff
@@ -38,65 +36,41 @@ export default function Header() {
       <Link to="/">
         <img src={"/icon/header/Logo.jpeg"} height={27} />
       </Link>
-      {authState === "valid" ? (
-        <Nav>
-          <NavButton
-            onClick={() => {
-              deleteSsoToken();
-              queryClient.invalidateQueries(["auth"]);
-              navigate("/");
-            }}
-          >
-            <img src={"/icon/header/Logout.svg"} />
-            로그아웃
-          </NavButton>
-          <NavButton
-            onMouseEnter={() => setShowApply(true)}
-            onMouseLeave={() => setShowApply(false)}
-          >
-            <img src={"/icon/header/Apply.svg"} />
-            지원페이지
-            {showApply && (
-              <ApplyList>
-                <NavLink to={"/recruiting/1"}>루키</NavLink>
-                <NavLink to={"/recruiting/2"}>디자이너</NavLink>
-              </ApplyList>
-            )}
-          </NavButton>
-          <NavLink to="/announcement">
-            <img src={"/icon/header/Alarm.svg"} />
-            공지사항
-          </NavLink>
-        </Nav>
-      ) : (
-        <Nav>
-          <NavButton
-            onClick={() => {
-              tryLogin("home");
-            }}
-          >
-            <img src={"/icon/header/Login.svg"} />
-            로그인
-          </NavButton>
-          <NavButton
-            onMouseEnter={() => setShowApply(true)}
-            onMouseLeave={() => setShowApply(false)}
-          >
-            <img src={"/icon/header/Apply.svg"} />
-            지원페이지
-            {showApply && (
-              <ApplyList>
-                <NavButton onClick={() => tryLogin(1)}>루키</NavButton>
-                <NavButton onClick={() => tryLogin(2)}>디자이너</NavButton>
-              </ApplyList>
-            )}
-          </NavButton>
-          <NavLink to="/announcement">
-            <img src={"/icon/header/Alarm.svg"} />
-            공지사항
-          </NavLink>
-        </Nav>
-      )}
+      <Nav>
+        {authState === "valid" ? (
+          <>
+            <NavButton
+              onClick={() => {
+                deleteSsoToken();
+                queryClient.invalidateQueries(["auth"]);
+                navigate("/");
+              }}
+            >
+              <img src={"/icon/header/Logout.svg"} />
+              로그아웃
+            </NavButton>
+          </>
```

Comment:
```suggestion
          <NavButton
            onClick={() => {
              deleteSsoToken();
              queryClient.invalidateQueries(["auth"]);
              navigate("/");
            }}
          >
            <img src={"/icon/header/Logout.svg"} />
            로그아웃
          </NavButton>
```

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md 요거 켜두면 좋슴다